src/main/java/mil/dds/anet/config/SecurityConfig.java

```diff
@@ -106,11 +106,30 @@ public SecurityFilterChain soapSecurityFilterChain(HttpSecurity http) throws Exc
 
   @Bean
   @Order(20)
+  public SecurityFilterChain graphQLWebServicveFilterChain(HttpSecurity http) throws Exception {
+    http.securityMatcher("/graphqlWebService/**") // Apply only to GraphQL endpoints
+        .authorizeHttpRequests(auth -> auth.anyRequest().permitAll());
+
+    // Stateless, disable session management
+    http.sessionManagement(
+        session -> session.sessionCreationPolicy(SessionCreationPolicy.STATELESS));
+    // Disable CSRF
+    http.csrf(AbstractHttpConfigurer::disable);
+    // Configure CSP
+    http.headers(
+        headers -> headers.contentSecurityPolicy(csp -> csp.policyDirectives(soapCsp.toString())));
```

Comment:
Perhaps rename `soapCsp` to `webServiceCsp` to indicate that is is not just used for SOAP?